low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -70,40 +70,25 @@ SSPL_LL_SETTING:
       - CPUFaultSensor
 
 RABBITMQINGRESSPROCESSOR:
-   virtual_host: SSPL
-   queue_name: actuator-req-queue
-   exchange_name: sspl-in
-   routing_key: actuator-req-key
-   username: sspluser
-   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
-   primary_rabbitmq_host: localhost
+   consumer_id: sspl_actuator
+   consumer_group: cortx_monitor
+   message_type: requests
+   offset: latest
 
 RABBITMQEGRESSPROCESSOR:
-   virtual_host: SSPL
-   queue_name: sensor-queue
-   exchange_name: sspl-out
-   routing_key: sensor-key
-   ack_queue_name: sensor-queue
-   ack_exchange_name: sspl-out
-   ack_routing_key: sensor-key
-   username: sspluser
-   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
    message_signature_username: sspl-ll
    message_signature_token: ALOIUD986798df69a8koDISLKJ282983
    message_signature_expires: 3600
-   iem_route_addr:
-   iem_route_exchange_name: sspl-out
-   primary_rabbitmq_host: localhost
    limit_consul_memory: 50000000
+   producer_id: sspl-sensor
+   message_type: alerts
+   method: sync
 
 LOGGINGPROCESSOR:
-   virtual_host: SSPL
-   queue_name: iem-queue
-   exchange_name: sspl-in
-   routing_key: iem-key
-   username: sspluser
-   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
-   primary_rabbitmq_host: localhost
+   consumer_id: sspl_in
+   consumer_group: cortx_monitor
+   message_type: IEM
+   offset: earliest
 
 RABBITMQCLUSTER:
```

Comment:
can remove this right ?

Reply:
We have separate (sub)task for removing rabbitmq references. That's why  I am not changing anything related to that in this PR. This PR is just for integrating message bus functionality.

Reply:
ok